CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ All notable changes to this project will be documented in this file.
 ## [2.4.1] - 2023-10-13
 
 ### Added
+- Added a `check_surface_based_positions` positions method for verifying antenna
+positions are near surface of whatever celestial body their positions are referenced to
+(either the Earth or Moon, currently).
```

Comment:
These should go under the `Unreleased` section.

Reply:
Whoops -- my brain saw `Unreleased` and completely skipped over the version right beneath it...

Reply:
Fixed!